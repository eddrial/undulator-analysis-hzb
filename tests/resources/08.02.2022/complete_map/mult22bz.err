 error report for the data set mult22bz.dvm        
--------------------------------------------
      
         127   data points thrown away to get a
         monotonically increasing data set
 the truncated data set has only   1591 data points 
        out of totally   2558 data points
 the derivative at  0.45722E+03 and  0.51215E+03 is zero 
 the derivative at  0.56920E+03 and  0.60046E+03 is zero 
 the derivative at  0.60046E+03 and  0.65286E+03 is zero 
 the derivative at  0.70948E+03 and  0.76652E+03 is zero 
 the derivative at  0.98878E+03 and  0.10201E+04 is zero 
 the derivative at  0.10201E+04 and  0.10433E+04 is zero 
 the derivative at  0.12398E+04 and  0.13234E+04 is zero 
 the derivative at  0.13234E+04 and  0.13547E+04 is zero 
 the derivative at  0.14122E+04 and  0.14367E+04 is zero 
 the derivative at  0.16615E+04 and  0.17202E+04 is zero 
 the derivative at  0.17734E+04 and  0.18546E+04 is zero 
 the derivative at  0.20527E+04 and  0.21060E+04 is zero 
 the rms value of the residuals is  0.17948E-02
 the maximum residual is  0.56000E-02
 the rms value of the lambda/2-filtered data is  0.13563E-02
 the maximum value of the lambda/2-filtered data is  0.89325E-02
