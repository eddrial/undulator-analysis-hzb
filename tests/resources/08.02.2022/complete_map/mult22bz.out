mult22bz.dvm
            

 names of parameters as follows : 
b(1) == amplitude, 1st harm.
b(2) == amplitude, 3rd harm.
b(3) == amplitude, 5rd harm.
....                        
b(n) == amplitude, n^th harm.
b(n+1) == lambda, mm.
b(n+2) == x0, 1st harm, mm.
b(n+3) == x0, 3rd harm, mm.
b(n+4) == x0, 5th harm, mm.
....
b(nparm) == x0, n^th harm, mm.

 parameters:
   1 -7.2072507E-03
   2 -4.3958298E-05
   3 -7.0509566E-05
   4 -4.5304664E-06
   5 -1.3314625E-06
   6  5.5968651E+01
   7  1.0456299E+03
   8  9.8353986E+02
   9  1.0746444E+03
  10 -1.9383764E+01
  11  9.0190058E+00
 number of iterations:   14
     sumsq:  5.12488E-03
       rms:  1.79476E-03  dimension of magnetic field
       rms:  2.49021E+01  percent of b_eff
   rms_ext:  1.76541E+01  percent of b_eff
 B_eff =   7.2073E-03
 B_max =  -7.1284E-03
 K_eff =    0.038
 lamda_1 (nm) =      2.54209

        eigenvectors in parameter space

 error                         direction
    47.45,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
    17.93,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00
     4.31,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00
     1.61,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00
     0.09,  0.00  0.00  0.00  0.00  0.00 -0.05  1.00  0.00  0.00  0.00  0.00
     0.01,  0.00  0.00  0.00  0.00  0.00  1.00  0.05  0.00  0.00  0.00  0.00
     0.00,  0.67 -0.19  0.11  0.60 -0.39  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.67  0.35  0.31 -0.49  0.31  0.00  0.00  0.00  0.00  0.00  0.00
     0.00, -0.28  0.64  0.53  0.11 -0.47  0.00  0.00  0.00  0.00  0.00  0.00
     0.00, -0.17 -0.24  0.65  0.38  0.59  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.04  0.61 -0.44  0.50  0.42  0.00  0.00  0.00  0.00  0.00  0.00
   
  calculated kicks for trajectory compensation: 
 kick_1:  -0.13836 Tmm
 kick_2:   0.13836 Tmm
      
 the rms value of the second derivative is  0.00000E+00
 the maximum value of the second derivative is  0.00000E+00
      
 the maximum transverse movement of the electron in mu_meter
                  (peak to peak) is  0.63146E+01
      
 the truncated data set has only   1591 data points 
      
 the rms value of the residuals is  0.17948E-02
 the maximum residual is  0.56000E-02
      
 the rms value of the lambda/2-filtered data is  0.13563E-02
 the maximum value of the lambda/2-filtered data is  0.89325E-02
      
 the rms phase error (all data) is    1.279 deg.
 the maximum phase error (all data) is    3.923 deg.
      
 the rms phase error (extrema) is    4.055 deg.
 the maximum phase error (extrema) is    9.696 deg.
      
 the Walker-rms phase error (all data) is    1.281 deg.
 the Walker-maximum phase error (all data) is    4.022 deg.
      
 the Walker-rms phase error (extrema) is    1.323 deg.
 the Walker-maximum phase error (extrema) is    3.217 deg.
  
  phase: straight line subtracted
 -----------------------------------------
 total phase error             1.278806    
 local phase error            0.5605574    
 slowly varying phase error  NaN           
 oscillatory phase error     NaN           
 -----------------------------------------
